app/src/main/java/com/github/mateo762/myapplication/search/SearchActivity.kt

```diff
@@ -30,18 +36,37 @@ class SearchActivity : BaseActivity() {
         layoutManager.orientation = LinearLayoutManager.VERTICAL
         binding.recyclerView.layoutManager = layoutManager
 
-        val items = listOf(
-            SearchItem("Pedro", "Son of Harry"),
-            SearchItem("Alice", "Married to Bob"),
-            SearchItem("Peter", "Married to Mary"),
-            SearchItem("Elisabeth", "Mother of Pascal"),
-            SearchItem("Charlotte", "Daughter of Boris")
-        )
+        println("Loaded the Search Activity")
 
-        adapter = SearchViewAdapter(items)
+        //TODO: Retrieve a list of users from Firebase
+        val database = FirebaseDatabase.getInstance()
+        val usersRef = database.getReference("usernames")
 
+        val users = mutableListOf(SearchItem("User", "Description"))
+        adapter = SearchViewAdapter(users)
         binding.recyclerView.adapter = adapter
 
```

Comment:
So here, as in the PR #98, if I get it right you're displaying all the search items in the recycler view, right?